site/content/docs/cli/command-line-reference.md

```diff
@@ -66,6 +66,66 @@ between environments.
 - `--verbose` or `-v`: Always show the full logs of the checks.
 
 
+## `npx checkly pw-test`
+
+Run Playwright tests with Checkly monitoring features. Records test sessions automatically.
+
+```bash
+npx checkly pw-test [checkly flags] -- [playwright flags]
+```
+
+Use `--` to separate Checkly flags from Playwright flags.
+
+### Basic examples
+
+Run tests in multiple browsers:
+```bash
+npx checkly pw-test -- --project=chromium --project=firefox
+```
+
+Run tests by pattern:
+```bash
+npx checkly pw-test -- --grep="login"
+```
+
+Exclude tests by pattern:
+```bash
+npx checkly pw-test -- --grep-invert="skip|todo"
+```
+
+### Creating monitors
```

Comment:
This is interesting. Maybe 1-2 sentences about the `--create-check` would help out here? What happens when I run the same `--create-check` multiple times? Is it smart enough not to duplicate things? How does this work?

Reply:
Not smart enough to not duplicate things yet, it appends the check again as a new one to your config.

Reply:
This is then definitely worth calling out. 😅 I could see myself stumbling into a gazillion created checks.